cookbooks/cdo-varnish/test/shared/shared.rb

```diff
@@ -489,13 +489,15 @@ def does_not_strip_cookies_from_png_in_path(path)
         end
       end
 
-      it 'Strips cookies from the penultimate dance level' do
-        assert strips_session_specific_cookies_from_request? '/s/dance/lessons/1/levels/12'
```

Comment:
We lost this `/dance/` (not 2019) test, no longer needed?

Reply:
There are lots of scripts and levels that we cache (and strip cookies from) so I changed this test to more of a spot check that some arbitrary levels in an arbitrary cached script behaves as expected.